test/WireMock.Net.Tests/FluentAssertions/WireMockAssertionsTests.cs

```diff
@@ -20,9 +23,21 @@ public WireMockAssertionsTests()
         {
             _server = WireMockServer.Start(Port);
             _server.Given(Request.Create().UsingAnyMethod())
-                            .RespondWith(Response.Create().WithStatusCode(200));
+                .RespondWith(Response.Create().WithStatusCode(200));
 
-            _httpClient = new HttpClient { BaseAddress = new Uri($"http://localhost:{Port}") };
+            _httpClient = new HttpClient {BaseAddress = new Uri($"http://localhost:{Port}")};
+        }
+
+        [Fact]
+        public async Task AtAbsoluteUrl_Should_NotThrowWhenAnyCallWasMadeAtAbsoluteUrl()
+        {
+            await _httpClient.GetAsync("anyurl");
+
+            Action act = () => _server.Should()
+                .HaveReceivedACall()
+                .AtAbsoluteUrl("http://localhost:42000/anyurl");
```

Comment:
Do not use a fixed port in unit-tests. 
`private const int Port = 42000;`

This can fail when running on local system or build-server because that one is already in use.

Use a random port.